cds_ils/migrator/api.py

```diff
@@ -95,3 +89,94 @@ def process_document_dump(data, source_type=None):
     except Exception as e:
         db.session.rollback()
         raise e
+
+
+def document_migration_report(
+    log_file, recid_list_file, output_filepath="/tmp/documents_report.csv"
```

Comment:
IMO it does not matter in this case, it is just for reporting file and /tmp is the only folder on each OS to which everyone can r/w so it simplifies things.
But I get your point, I could move it to cli.py maybe it would be better?

Reply:
Leave it like this ;)